src/compiler/config/validate-copy.ts

```diff
@@ -2,7 +2,19 @@ import { unique } from '@utils';
 
 import type * as d from '../../declarations';
 
-export const validateCopy = (copy: d.CopyTask[] | boolean, defaultCopy: d.CopyTask[] = []): d.CopyTask[] => {
+/**
+ * Validate a series {@link d.CopyTask}s
+ * @param copy the copy tasks to validate, or a boolean to specify if copy tasks are enabled
+ * @param defaultCopy default copy tasks to add to the returned validated list if not present in the first argument
+ * @returns the validated copy tasks. this function _may_ return an empty list if the first argument is:
```

Comment:
it looks like it's also possible if `copy` is `[]` and `defaultCopy` is either not passed or `[]` is passed, should we note that too?

Reply:
I went back and forth here. I didn't want to add _too_ much detail in the event that if the implementation changed, I was concerned we'd get out of sync with the JSDoc. Instead, [I decided to remove these details ](https://github.com/ionic-team/stencil/pull/3711/commits/a026828a70db8ce0802fbb00dd0e6d9d9720b491)and let the tests provide the documenation for us

Reply:
I think return type of `Foo[]` implies the possibility of `[]` as the returned value anyhow, so that's a 👍 from me